doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,107 +57,95 @@ Values are provided in kilobytes (KB).
 
    .. tab:: Thingy:53
 
       The following table lists memory requirements for samples running on the :ref:`Thingy:53 <programming_board_names>` (:ref:`thingy53_nrf5340 <zephyr:thingy53_nrf5340>`).
 
       +---------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                        |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                        |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
       +===============================================================+===============+===================+================+============+=============+=================================+
-      | :ref:`Weather Station <matter_weather_station_app>` (Debug)   |            64 |               766 |             16 |         48 |         894 |                             210 |
+      | :ref:`Weather Station <matter_weather_station_app>` (Debug)   |            64 |               709 |             16 |         48 |         837 |                             215 |
       +---------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Weather Station <matter_weather_station_app>` (Release) |            64 |               616 |             16 |         48 |         744 |                             184 |
+      | :ref:`Weather Station <matter_weather_station_app>` (Release) |            64 |               574 |             16 |         48 |         702 |                             188 |
       +---------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
    .. tab:: nRF7002 DK
 
       The following table lists memory requirements for samples running on the :ref:`nRF7002 DK <programming_board_names>` (:ref:`nrf7002dk_nrf5340_cpuapp <nrf7002dk_nrf5340>`).
 
       +------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                     |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                     |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

Comment:
```suggestion
      | Sample                                                     |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

Reply:
The tables will be updated later once again. I need to re-generate data.